setup.cfg

```diff
@@ -1,14 +1,138 @@
-[bdist_wheel]
-universal = 1
-
-# See the docstring in versioneer.py for instructions. Note that you must
-# re-run 'versioneer.py setup' after changing this section, and commit the
-# resulting files.
-
-[versioneer]
-VCS = git
-style = pep440
-versionfile_source = drms/_version.py
-versionfile_build = drms/_version.py
-tag_prefix = v
-parentdir_prefix = drms-
+[metadata]
+name = drms
+provides = drms
+description = "Access HMI, AIA and MDI data with Python"
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = The SunPy Community
+author_email = sunpy@googlegroups.com
+license = BSD 2-Clause
+license_file = LICENSE.rst
+url = https://sunpy.org
+edit_on_github = True
+github_project = sunpy/drms
+platform = any
+keywords = solar physics, solar, science, data
+classifiers =
+  Development Status :: 5 - Production/Stable
+  Intended Audience :: Science/Research
+  License :: OSI Approved :: BSD License
+  Natural Language :: English
+  Operating System :: OS Independent
+  Programming Language :: Python
+  Programming Language :: Python :: 3
+  Programming Language :: Python :: 3.7
+  Programming Language :: Python :: 3.8
+  Topic :: Scientific/Engineering :: Astronomy
+  Topic :: Scientific/Engineering :: Physics
+
+[options]
+zip_safe = False
+python_requires = >=3.7
+packages = find:
+include_package_data = True
+setup_requires =
+  setuptools_scm
+install_requires=
+  numpy
+  pandas
+
+[options.extras_require]
+tests =
+  pytest-astropy
+  # If you plan to run the tests locally this has to be installed.
```

Comment:
it does?

Reply:
Yes, it goes to get the version and errors if setup_scm isn't installed.

Reply:
Nope, it was a me issue.